apps/toolkit-app/pages/index.tsx

```diff
@@ -0,0 +1,272 @@
+import { Suspense } from "react"
+import Image from "next/image"
+import Link from "next/link"
+import Layout from "app/core/layouts/Layout"
+import { useCurrentUser } from "app/core/hooks/useCurrentUser"
+import logout from "app/auth/mutations/logout"
+import logo from "public/logo.png"
+import { invoke } from "@blitzjs/rpc"
```

Comment:
Should be replaced with useQuery after https://github.com/blitz-js/blitz/pull/3302 is merged